config/kubernetes/sig-docs/teams.yaml

```diff
@@ -313,7 +312,6 @@ teams:
     description: Write access to the website repo
     members:
     - aisonaku # L10n: Russian
-    - alexbrand # L10n: Spanish
```

Comment:
👋 @raelga .
Do you want to add `@electrocucaracha` to this section?

Reply:
Thanks @kbhawkey for pointing that out! 

At this stage, we are not using any of the capabilities of the `website-mantainers` role as we don't have any branching strategy. But up to you @electrocucaracha!